### DISCUSSION TITLE
How about pull request review difficulty?

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#103] [General]
첫 pull request를 다들 잘 해주셨는데요?

개인적으로 생각하는 난이도가 어떤지
그리고 review 방식에 대해 다른 방향성에 대한 의견을 들어보려고 합니다.

아래 카테고리 안에서 한번 생각해 보신 후에 자유롭게 의견 주시면 좋을 것 같습니다.
향후 pull request review 할 때 반영하도록 하겠습니다.

- 총 8개의 pull request를 리뷰 하는데 어느 정도로 빡세다고 느끼시는지?
- 47시간 안에 review 하는 기간의 적정성

**Comment by momoci99** [comment]
음..  PR을 리뷰할 수 있는 시간이 좀 더 여유있었으면 좋겠습니다. 하루나 이틀정도 + 되면 좋을 것 같아요. 제 예상보다 8개의 PR은 빡빡하더군요 ㅜㅜ

**Reply by evelon** [reply]
저도 삼 일 정도는 필요하지 않나 생각이 들었습니다.

**Comment by momoci99** [comment]
다른분들 의견도 궁금합니당..

**Comment by wooyaggo86** [comment]
저에게는 적정한데, 백수라 다른분들과 비교가 어려울 것 같네요..
다음 주에 취업하면 의견 드리겠습니다🤣

**Comment by hi-hyein** [comment]
피드백이 좀 늦었습니다.
2021때보다 분량은 줄었지만.. 인원이 두배로 늘어서ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ
다른 분들의 pr을 리뷰하는 시간을 하루정도 더 추가하는 것도 좋을 것 같습니다~

**Comment by jongfeel** [comment]
다음 모임 부터는 모임 3일전 pull request를 주고 3일간 리뷰하는 시간으로 결정하겠습니다.
